package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@oclif/core": "^1.7.0",
     "@salesforce/core": "^3.15.5",
+    "@salesforce/kit": "^1.5.41",
```

Comment:
Heya @mdonnalley why is this bringing kit in?

I don't see it being used in any of the changes.

Reply:
It's being elsewhere in the plugin though. 

It never caused an issue since kit is almost guaranteed to be somewhere in the dependency tree. But I thought it'd be wise to explicitly add it